src/stats/stats_printer.cpp

```diff
@@ -318,12 +378,31 @@ void print_netstat_like(socket_stats_t* p_si_stats, mc_grp_info_t* , FILE* file,
 {
 	static const int MAX_ADDR_LEN = strlen("123.123.123.123:12345"); // for max len of ip address and port together
 	char process[FILE_NAME_MAX_SIZE];
+	uint32_t rxq, txq;
+	const char *socket_type = NULL;
 
 	if(! p_si_stats->inode) return; // shmem is not updated yet
 
 	fprintf(file, "%-5s %-9s ", to_str_socket_type_netstat_like(p_si_stats->socket_type), p_si_stats->b_is_offloaded ? "Yes" : "No");
 
 	//
+	// SOCK_STREAM does not use mixed VMA+OS statistic
+	//
```

Comment:
We will not add the OS UDP stats for Recv-Q and Send-Q for the following reasons:
1. User can run 'netstat -tpn' and have the same output for OS connections
2. libvma is very performance sensitive (that includes vma_stats as well) and we may encounter cases of thousands of OS sockets (non-offloaded)

Specific remarks about existing implementation
1. For every inode the file /proc/net/udp is read from scratch. For thousands of connections it is not scalable. As an alternative the file could have been read once into some data structures and then accessed in a more time-saving manner.
2. In case we print OS connections from vma_stats in the future - we will separate the offloaded statistics from the OS statistics by printing them in 2 different lines (remark: as written now - offloaded and OS statistics are summarized).

OS statistics code is highly appreciated, but for now please remove related calls (including get_proc_net_value() function).